src/const/actions.js

```diff
@@ -0,0 +1,7 @@
+const actions = {
```

Comment:
It cool for now.
But for the future, we would like to suggest you to use `duck` app structure. It is awesome)
Here is it: https://github.com/erikras/ducks-modular-redux